app/models/paper.rb

```diff
@@ -1,10 +1,19 @@
 class Paper < ApplicationRecord
   acts_as_taggable
+  acts_as_taggable_on :biases
+  ActsAsTaggableOn.strict_case_match = true
 
   has_and_belongs_to_many :authors
   has_many :lists, through: :references
   has_many :references, dependent: :destroy
 
   accepts_nested_attributes_for :authors, reject_if: proc { |attributes| attributes['name'].blank? }
   validates_associated :authors
+  validate :bias_list_inclusion
+
+  def bias_list_inclusion
+    bias_list.each do |bias|
+      errors.add(bias,"is not valid") unless %w(selection performance detection attrition reporting small\ sample\ size).include?(bias)
```

Comment:
Let's break `%w(selection performance detection attrition reporting small\ sample\ size)` into its own private method, perhaps called `valid_biases`. Then you could do:

```ruby
def allowed_biases
  invalid_biases = bias_list - valid_biases
  invalid_biases.each {|b| errors.add(b, 'is not valid') }
end
```